src/redux/modules/counter.js

```diff
@@ -1,17 +1,15 @@
-import createReducer from 'utils/createReducer'
+import { createAction,
```

Comment:
You mean having mutiple named imports on different lines? Yes, I do, because the lists can get to be long in some cases, so I just adopt the rule of new-lining more than one, though I'm not always consistent when it's just two.